### PR TITLE
MM-26620 Add system console setting to disable ask the community link

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Adds a system console setting to disable ask the community link in help guide dropdown 

PR is on top of https://github.com/mattermost/mattermost-webapp/pull/5843. Changes for this PR are at https://github.com/mattermost/mattermost-webapp/commit/0a57fda03ad8facf4c880b29a5a9da7a447e174a
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-26620

#### Related Pull Requests
https://github.com/mattermost/mattermost-redux/pull/1188

#### Screenshots
Untitled

### CODE REVIEW

**Comment by jgilliam17** [comment]
@jasonblais Can you please provide some clarification on capitalization for "community"
Help text shows "Ask the Community" and the menu option is  "Ask the community".  We should make it consistent. 
Screen Shot 2020-07-13 at 10 41 54 AM
Screen Shot 2020-07-13 at 11 05 21 AM

**Comment by jasonblais** [comment]
@jgilliam17 Good catch, it should be consistent.

@sudheerDev Can you help update the help text to read: `When true, "As the community" link appears ...`

**Comment by jasonblais** [comment]
@jgilliam17 Text updated :

File: sass/components/_channel-header__icon.scss [hunk 1]
```diff
@@ -52,4 +52,8 @@
 .channel-header__icon--wide {
     width: auto;
     padding: 0 6px;
-}
\ No newline at end of file
+}
+
+.userGuideHelp {
+    display: inline-table;
```
Comment (deanwhillier) [inline sass/components/_channel-header__icon.scss:58]: Hmm, this is really hacky, but I don't see a better way short of replacing this with the new upcoming menu common component. 😛

**Pull request merged**